api/app/src/routes/internal.ts

```diff
@@ -24,4 +25,22 @@ router.post(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * POST /internal/mapi
```

Comment:
should we add a delete endpoint while we're at it?

could be useful to have both for internal OPS

Reply:
💯